studies/india_districts/etl.py

```diff
@@ -160,14 +160,22 @@
 }
 
 column_ordering_v4  = [
-    'Patient Number',
-    'Date Announced',
-    'Detected District',
-    'Detected State',
-    'Current Status',
-    'Status Change Date',
-    'Num cases'
-]
+    'patient_number',
+     'date_announced',
+     'detected_district',
+     'detected_state',
+     'current_status',
+     'status_change_date',
+     'num_cases'
+ ]
+
+def download_data(data_path: Path, filename, base_url='https://api.covid19india.org/csv/latest/'):
```

Comment:
duplicate of line 180?

Reply:
woops sorry, think this should be fixed now